app/presenters/licences/view-licence-points.presenter.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+/**
+ * Formats the licence and related points data for the view licence points page
+ * @module ViewLicencePointsPresenter
+ */
+
+const PointModel = require('../../models/point.model.js')
+
+/**
+ * Formats the licence and related points data for the view licence points page
+ *
+ * @param {object} data - The licence and related points data returned by `FetchLicencePointsService`
+ *
+ * @returns {object} licence and points data needed by the view template
+ */
+function go (data) {
+  const { licence, points } = data
+
+  const licencePoints = _formatLicencePoints(points)
+
+  return {
+    id: licence.id,
+    licencePoints,
+    licenceRef: licence.licenceRef,
+    pageTitle: 'Licence abstraction points'
+  }
+}
+
+function _formatLicencePoints (points) {
+  return points.map((point) => {
+    const pointInstance = PointModel.fromJson(point)
+
+    return {
```

Comment:
SonarCloud is complaining about the functions complexity with the added ternary operations going on. I couldn't think of a better way to do this, but I'm open to suggestions.

Reply:
We could try using the nullish coalescing operator `??` here:

```js
    return {
      bgsReference: pointInstance.bgsReference ?? '',
      category: pointInstance.category ?? '',
      // etc...
    }
```

If the first thing isn't `null` or `undefined` then it gives us the first thing, otherwise it gives us the second thing, so effectively shorthand for what we've got here. It may keep SonarCloud happier, or it may not!